rj_gameplay/rj_gameplay/tactic/pass_tactic.py

```diff
@@ -81,7 +81,7 @@ def get_requests(
 
         role_requests: tactic.RoleRequests = {}
 
-        passer_request = role.RoleRequest(role.Priority.HIGH, True, self.passer_cost)
+        passer_request = role.RoleRequest(role.Priority.HIGH, True, self.Passer_cost)
         role_requests[self.pivot_kick] = [passer_request]
         receive_request = role.RoleRequest(role.Priority.HIGH, True, self.receiver_cost)
```

Comment:
Based on understanding, it requests receive skill and capture skill simultaneously. so when robot 1 passes the ball, robot 7 moves toward the kicker, and it makes it difficult for the receiver to settle the ball, I believe. Capture skill should be applied only when the ball gets closer to the receiver??

Reply:
this is what does happen, kind of, if you check the receive action its based off of the speed of the ball, so when the ball is slow the capture starts. I am also changing it so that if the robot has ball sense it starts capture.

Reply:
Great! Now I understand what was happening!